website/content/docs/runner/index.mdx

```diff
@@ -37,6 +37,102 @@ to scale further.
+When this runner profile is used, Waypoint will only generate on-demand runners using a runner whose labels match those
+specified in the runner profile.
+
+```terminal
+% waypoint runner profile set -name=docker-hashicorp-dev -plugin-type=docker -target-runner-label=org=hashicorp -target-runner-label=env=dev
+✓ Runner profile created
+```
+
+```shell-session
+                  NAME                  | PLUGIN TYPE |               OCI URL                |              TARGET RUNNER              | DEFAULT
+----------------------------------------+-------------+--------------------------------------+-----------------------------------------+----------
+  docker-hashicorp-dev                  | docker      | hashicorp/waypoint-odr:latest        | labels: {"env":"dev","org":"hashicorp"} |
+```
+
+In the `waypoint.hcl` file, you can specify the runner profile to use with the `runner` stanza. In this example, a user
+variable "org" and the built-in `workspace` variable are interpolated into the runner profile name, so that at the time
+of execution of remote operations, Waypoint uses the desired runner profile.
+
+```hcl
+project = "example-nodejs"
+
+app "example-nodejs" {
+  runner {
+    profile = "docker-${var.org}-${workspace.name}"
+  }
+
+  build {
+    # ...
+  }
+
+  deploy {
+    # ...
+  }
+
+  release {
+    # ...
+  }
+}
+
+variable "org" {
+  type = string
+}
+```
+
+An operation which would use the runner profile created in this example could be initiated with `waypoint build -w=dev -var=org=hashicorp -local=false`.
+
+### Runner Targeting Advantages
+
+The example above demonstrates basic usage of runner targeting. However, in a production environment, runner targeting
+may be essential to achieving high availability. Multiple runners could be deployed with the same labels, and then a
+runner profile which targets runners with those labels would "round robin" between the available runners. If one goes down,
+as long as another with the same labels is still online, remote operations will continue to work.
+
+Also evidenced in the example is runner targeting for multi-env operations. In the example, the workspace of the operation,
+dev, is interpolated into the runner profile name dynamically at operation runtime. Additional runners & runner profiles
+could be created for a "test" or "prod" environment, for example, so that at runtime a runner to deploy to the right environment
+is used.
+
+Runner targeting is also useful for workflows where you deploy to multiple clusters in a platform. For example, if you have
+a runner in one Kubernetes cluster, that runner could potentially only perform remote operations within that cluster. However,
+if there were a runner in multiple Kubernetes clusters, you would have the ability to deploy to any of the clusters with
+Waypoint remote operations by targeting the appropriate runners in each cluster.
+
```

Comment:
This is something you can likely configure in your editor, but for code as well as documentation, we want to keep the lines as close to 80 characters long as possible, and I think this is over that?